tests/db_utils.py

```diff
@@ -1,5 +1,16 @@
 import os
 import psycopg2
+from psycopg2.extensions import quote_ident
+
+def ensure_environment_variables_set():
+    missing_envs = [x for x in [os.getenv('TAP_POSTGRES_HOST'),
+                                os.getenv('TAP_POSTGRES_USER'),
+                                os.getenv('TAP_POSTGRES_PASSWORD'),
+                                os.getenv('TAP_POSTGRES_PORT'),
+                                os.getenv('TAP_POSTGRES_DBNAME')] if x is None]
+    if len(missing_envs) != 0:
+        #pylint: disable=line-too-long
+        raise Exception("set TAP_POSTGRES_HOST, TAP_POSTGRES_DBNAME, TAP_POSTGRES_USER, TAP_POSTGRES_PASSWORD, TAP_POSTGRES_PORT")
```

Comment:
Not a huge deal, but you can replace this message with a format string of "set {missing_envs}".format(missing_envs=missing_envs).

Reply:
mmm yeah that would be better. I'll rework this.

Reply:
Updated this, and calling it in all tests now.